src/magnify/apps/core/models.py

```diff
@@ -15,3 +16,98 @@ class Meta:
 
     def __str__(self):
         return self.username
+
+class Meeting(models.Model):
+    """Model for one meeting or a collection of meetings defined reccursively"""
+    name = models.CharField(max_length=100)
+
+    # Start and end are the same for a single meeting
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
+
+    # Set to True if there is a meeting on that day
+    monday = models.BooleanField(default=False)
+    tuesday = models.BooleanField(default=False)
+    wednesday = models.BooleanField(default=False)
+    thursday = models.BooleanField(default=False)
+    friday = models.BooleanField(default=False)
+    saturday = models.BooleanField(default=False)
+    sunday = models.BooleanField(default=False)
+
+    start_Time = models.TimeField()
```

Comment:
```suggestion
    start_time = models.TimeField()
```
But I'm not sure if this field is still necessary now that the `start` field is a datetime?

Reply:
In the documentation for example:
```
We should be able to define a meeting taking place once, starting on a predefined date and time, for example June 13, 2022, and expected to last 2h. In this case, the instance would take the following values:
start = ...
end = ...
...